charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+# Data Portal
+
+
+## Description
+The Data Portal provides the overall web-site for the HCA DCP with entry point from https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
+## Objective
+Implementation of the Data Coordination Platform (DCP) website experience. Provide the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, providing file/bundle search on multiple clouds at PB scale, with strong public APIs.
+
+## In-scope
+### General
+* UX, visual and branding design with input from HCA leadership (GG, scientific steering committee, etc.)
+* Participation in the HCA-wide standards for Continuous Integration (CI), Continuous Delivery (CD), system logging via Google Analytics, etc.
+* Production operation of the overall Data Portal web-site (see exclusions in Out-of-scope section)
+* Implementation performance scaling
+* DevSecOps compliance as required by the HCA Security Team
+* DevSecOps features for FISMA moderate deployments (authZ/N, logging, auditing, etc) by leveraging other projects (if this becomes a future requirement)
+* Community engagement
+   * Triage and integration of feature requests into the work queue 
+   * Support plan for escalation of user reported bugs
+### Data Portal
+* Select and deploy content management system
+* Integrate Data Browser and Data Portal UIs into a seamless UX and provide link to other groups' applications (e.g. Ingest UI)
+* Frequent updates of summary statistics (cells, organs, donors, projects, labs, etc.) on Home Page
+* Coordinate development of user guide content (e.g. “Learn”) from multiple contributors
+* Coordinate development of developer documentation content (e.g. “Develop”) from multiple contributors. Includes API documentation and development guides.
+* Click through to Data Browser from search box and human body image
+* Registration form for tertiary portals and list of portals available with click through to hosting web site (e.g. Dockstore)
+* Integration with Zendesk help desk
+* Integration with Vanilla Forums (discussion forum for developers)
+* Integration with Canny (to capture user feedback)
+### Data Browser
+* Create (or use Data Store provided) indexes as necessary for files in the Data Store, using public DSS APIs to read the metadata
```

Comment:
This is the first mention of the Data Storage Service. Define acronym here as `Data Storage Service (DSS)` for later use.